src/main/webapp/volunteer/registration.html

```diff
@@ -21,6 +21,21 @@
     <title>Registration</title>
 </head>
 <body>
+<h1>Welcome, volunteer!</h1>
+<h2>Please register using the form below.</h2>
+
+<form id="volunteer-registration-form" method="POST" action="/volunteer-registration">
+    <label for="firstname">First Name</label>
+    <input name="firstname" id="firstname" required minlength="2" maxlength="50"/>
```

Comment:
In future we might define these constants somewhere else, but for this PR I think it's fine.